WooCommerce/Classes/POS/Models/PointOfSaleSettingsService.swift

```diff
@@ -0,0 +1,95 @@
+import Foundation
+import struct Yosemite.SiteSetting
+import enum Yosemite.SettingAction
+import enum Yosemite.Plugin
+import class Yosemite.PluginsService
+import Observation
+
+@Observable final class PointOfSaleSettingsService {
+    private(set) var receiptStoreName: String?
+    private(set) var receiptStoreAddress: String?
+    private(set) var receiptStorePhone: String?
+    private(set) var receiptStoreEmail: String?
+    private(set) var receiptRefundReturnsPolicy: String?
+    private(set) var isLoading: Bool = false
+    private(set) var shouldShowReceiptInformation: Bool = false
+
+    private var siteID: Int64 {
+        ServiceLocator.stores.sessionManager.defaultSite?.siteID ?? 0
+    }
+
+    var storeName: String {
+        guard let site = ServiceLocator.stores.sessionManager.defaultSite else {
+            return Localization.storeNotSet
+        }
+        return site.name
+    }
+
+    var storeAddress: String {
+        SiteAddress().address
```

Comment:
❓ Similar question to the store name, how is the store address used in POS? I'm just not sure how this value is affecting the POS flow.

And similar nit: `SiteAddress()` is using `ServiceLocator.selectedSiteSettings.siteSettings` as the default value, would be great DI'ing them.